package.json

```diff
@@ -4,10 +4,10 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "start": "react-scripts start --NODE_ENV=production && node index.jsx",
+    "start": "react-scripts start --NODE_ENV=production&&node index.jsx",
```

Comment:
Are the spaces not required on the ampersands? It should be AFAIK

Reply:
Yes they are